setup.py

```diff
@@ -9,10 +9,11 @@
 
 INSTALL_REQUIRES = ['aenum~=3.1.15',
                     'datamodel_code_generator[http]~=0.25.0',
-                    'paho-mqtt~=1.6.1',
+                    'paho-mqtt>=2.0.0',
```

Comment:
Could you please change it to paho-mqtt~=2.0.0 instead of paho-mqtt>=2.0.0 otherwise the code might not work if there is a major release.

Reply:
@sbanoeon Changed !